README.md

```diff
@@ -239,3 +239,28 @@ The `CSVREAD` function provided by the H2 database engine allows you to query an
 Like all dataflow operators in Nextflow, the operators provided by this plugin are executed asynchronously.
 
 In particular, data inserted using the `sqlInsert` operator is _not_ guaranteed to be available to any subsequent queries using the `fromQuery` operator, as it is not possible to make a channel factory operation dependent on some upstream operation.
+
+## Running Integration Tests
+
+To run the integration tests, you'll need to set up the following environment variables:
+
+### Databricks Integration Tests
```

Comment:
I don't we should highlight databricks in the readme. This is supposed to be a generic plugin

Reply:
It's more for running the tests

Reply:
I'm hoping to merge #10 later and make sure all of the integrations are working as expected.